README.developers.md

```diff
@@ -184,6 +186,80 @@ VTR also has a limited set of unit tests, which can be run with:
 $ make && make test
 ```
 
+### Running tests on presubmits via Kokoro
+
+Because of the long runtime for nightly and weekly tests, a kokoro job can be
+used to run these tests once a Pull Request (PR) has been made at
+https://github.com/verilog-to-routing/vtr-verilog-to-routing.
+
+Any pull request made by a contributor of the verilog-to-routing GitHub project
+on https://github.com/verilog-to-routing/ will get a set of jobs immediately.
+Non-contributors can request a contributor on the project add a label
+"kokoro:force-run" to the PR. Kokoro will then detect the tag, remove the tag,
+and then and issue jobs for that PR.  If the tag remains after being added,
+there may not be an available Kokoro runner, so wait.
+
+#### Re-running tests on Kokoro
+
+If a job fails due to an intermittent failure or a re-run is desired, a
+contributor can add the label "kokoro:force-run" to re-issue jobs for that PR.
+
+#### Checking results from Kokoro tests
+
+Currently there is not a way for an in-flight job to be monitored.
+
+Once a job has been completed, the stdout and the output files (e.g.
+parse\_results.txt) are available by following the "Details" link that appears
+on the PR.
+
+The stdout from run is available on the "Invocation Log" tab.  The log should
+be displayed, and can be downloaded via the "Download Full Log" button on the
+same tab.
+
+Output files (e.g. parse\_results.txt) can be found by following the "GCS"
+link on the "Invocation Details" tab.  Individual files can be downloaded by
+using the GCS browser.  If many files are desired, use "gsutil" to download
+the logs from GCS.
+
+#### Example of downloading logs from GCS
+
+An GCS example link from the "Invocation Details" tab:
+
+```
+https://pantheon.corp.google.com/storage/browser/vtr-verilog-to-routing/artifacts/prod/foss-fpga-tools/verilog-to-routing/upstream/presubmit/nightly/52/20200123-165906
```

Comment:
`corp.google.com` is here and below.
Externally the URL is different. Try using incongnito mode to figure out the public URL.

Reply:
Fixed